aten/src/ATen/native/CPUBlas.cpp

```diff
@@ -250,4 +262,160 @@ void gemm(
       transa, transb, m, n, k, alpha, a, lda, b, ldb, beta, c, ldc);
 }
 
+DEFINE_DISPATCH(axpy_stub);
+
+void axpy(int64_t n, double a, const double *x, int64_t incx, double *y, int64_t incy) {
+  if(n == 1)
+  {
+    incx = 1;
+    incy = 1;
+  }
+  #if AT_BUILD_WITH_BLAS()
+  if( (n <= INT_MAX) && (incx <= INT_MAX) && (incy <= INT_MAX) )
+  {
+    int i_n = (int)n;
+    int i_incx = (int)incx;
+    int i_incy = (int)incy;
+    daxpy_(&i_n, &a, x, &i_incx, y, &i_incy);
```

Comment:
this doesn't look correct. why does it always call into the function for `double`?